src/Reader/Json.php

```diff
@@ -102,4 +92,79 @@ protected function process(array $data)
         }
         return $data;
     }
+
+    /**
+     * Decode JSON configuration.
+     *
+     * Determines if ext/json is present, and, if so, uses that to decode the
+     * configuration. Otherwise, it uses zend-json, and, if that is missing,
+     * raises an exception indicating inability to decode.
+     *
+     * @param string $data
+     * @return array
+     * @throws Exception\RuntimeException for any decoding errors.
+     * @throws Exception\MissingExtensionException if neither ext/json nor
+     *     zend-json are available.
+     */
+    private function decode($data)
+    {
+        if (function_exists('json_decode')) {
```

Comment:
Ok, I like it, but now we lost zend-json fallback IF somebody doesn't have this extension and can't add it (_IF_ it is even possible to not have it now on PHP 5.6+)